docs/guides/python_chip_controller_building.md

```diff
@@ -255,429 +258,195 @@ the following stages:
     finished and the Python CHIP controller is now using only the IPv6 traffic
     to reach the device.
 
-### Step 6: Control application ZCL clusters.
+### Step 6: Control application clusters.
 
 For the light bulb example, execute the following command to toggle the LED
 state:
 
 ```
-chip-device-ctrl > zcl OnOff Toggle 1234 1 0
+await devCtrl.SendCommand(1234, 1, Clusters.OnOff.Commands.Toggle())
 ```
 
-**REPL Command:**
-`await devCtrl.SendCommand(1234, 1, Clusters.OnOff.Commands.Toggle())`
-
 To change the brightness of the LED, use the following command, with the level
 value somewhere between 0 and 255.
 
 ```
-chip-device-ctrl > zcl LevelControl MoveToLevel 1234 1 0 level=50
+commandToSend = LevelControl.Commands.MoveToLevel(level=50, transitionTime=Null, optionsMask=0, optionsOverride=0)
+await devCtrl.SendCommand(1234, 1, commandToSend)
 ```
 
-**REPL Command:**
-`await devCtrl.SendCommand(1234, 1, LevelControl.Commands.MoveToLevel(level=50, transitionTime=Null, optionsMask=0, optionsOverride=0))`
-
 ### Step 7: Read basic information out of the accessory.
 
 Every Matter accessory device supports a Basic Information Cluster, which
 maintains collection of attributes that a controller can obtain from a device,
-such as the vendor name, the product name, or software version. Use `zclread`
-command to read those values from the device:
+such as the vendor name, the product name, or software version. Use
+`ReadAttribute()` command to read those values from the device:
 
 ```
-chip-device-ctrl > zclread BasicInformation VendorName 1234 1 0
-chip-device-ctrl > zclread BasicInformation ProductName 1234 1 0
-chip-device-ctrl > zclread BasicInformation SoftwareVersion 1234 1 0
+attributes = [
+    (0, Clusters.BasicInformation.Attributes.VendorName),
+    (0, Clusters.BasicInformation.Attributes.ProductName),
+    (0, Clusters.BasicInformation.Attributes.SoftwareVersion),
+]
+await devCtrl.ReadAttribute(1234, attributes)
 ```
 
-**REPL Command:**
-`await devCtrl.ReadAttribute(1234, [(1, Clusters.BasicInformation.Attributes.VendorName)])`
-
-> Use the `zcl ? BasicInformation` command to list all available commands for
-> Basic Information Cluster.
->
 > In REPL, you can type `Clusters.BasicInformation.Attributes.` and then use the
 > TAB key.
 
 <hr>
 
 ## List of commands
```

Comment:
We now have a full API description from the ChipDeviceCtrl documentation here (https://project-chip.github.io/connectedhomeip-doc/testing/ChipDeviceCtrlAPI.html) - it's generated from the function comments.

I think this documentation was written before some of the functions were updated, and before some of the new commissioning stuff landed, so there are a few things in here that are a bit out of date. WDYT about just getting rid of this section and making a note that the devCtrl is a device controller and you can use any of the normal functions, and then just linking over to that documentation?